unlock-app/src/components/creator/CreatorLocks.js

```diff
@@ -16,10 +16,47 @@ export class CreatorLocks extends React.Component {
     const { showForm } = this.props
     this.state = {
       showDashboardForm: !!showForm,
+      editingLocks: {},
```

Comment:
This works, but I wonder if there would not be a way to isolate the lock "state" (editing/not editing) in its state? (basically only the lock knows whether it is being edited by having some kind of property on it). I think this would be better separation of concerns?